tests/components/element-card/element-card.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { ElementCard } from "src/components";
+import { EElementType, TElement } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper, mockBuilderState } from "src/util/test-helper";
+
+describe("element-card", () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    describe("onClick", () => {
+        it("should be in a focused state and show the edit element panel when clicked on", () => {
```

Comment:
As you have no control of external, it is better to keep the test cases more relevant to the context of the component. 

```suggestion
        it("should fire the onClick callback when clicked", () => {
```